service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsController.java

```diff
@@ -152,8 +148,7 @@ private void removeDocumentFromData(Map<String, Object> data) {
         ZonedDateTime zonedDateTime = ZonedDateTime.now(ZoneId.of("Europe/London"));
 
         var c2DocumentBundleBuilder = caseData.getTemporaryC2Document().toBuilder()
-            .author(userDetailsService.getUserName())
```

Comment:
are we sure about this, recently we refined story that says uploader name should be attached to user. There are 2 conflicting screenshots in story, please double check with bisiness/design

Reply:
checked with design, it was mistake in one of the screenshots and it should be kept it, with content updated from 'Author' -> 'Uploaded by'